### PR TITLE
[CLI] Move mint, accredit and nonAccreditLimit code to its corresponding module script

### PR DESCRIPTION
**Pull request by shuffledex**

### CODE REVIEW

File: CLI/commands/sto_manager.js [hunk 1]
```diff
@@ -702,7 +708,7 @@ async function usdTieredSTO_configure(currentSTO) {
         await common.sendTransaction(changeNonAccreditedLimitAction);
         break;
       case 4:
-        await changeNonAccreditedLimit.executeApp(tokenSymbol, 75);
+        await startCSV(tokenSymbol, 75, 'nonAccredited');
```
Comment (VictorVicente) [inline CLI/commands/sto_manager.js]: Change 'nonAccredited` to `nonAccreditedLimit`

File: CLI/commands/sto_manager.js [hunk 2]
```diff
@@ -728,6 +734,166 @@ async function usdTieredSTO_configure(currentSTO) {
...
+
+  let usdTieredSTO = new web3.eth.Contract(abis.usdTieredSTO(), gtmModules[0]);
+
+  console.log(`
+    --------------------------------------------------------
+    ----- Sending accreditation changes to blockchain  -----
+    --------------------------------------------------------
+  `);
+
+  for (let i = 0; i < distribData.length; i++) {
```
Comment (VictorVicente) [inline CLI/commands/sto_manager.js]: Why are there two `for` blocks?

**Review by VictorVicente — CHANGES_REQUESTED** [review]

File: CLI/commands/sto_manager.js (hunk 2 above)
Comment (shuffledex) [reply CLI/commands/sto_manager.js]: Interesting... I never touched this part of the code. Maybe a type error? or something intentional?

File: CLI/commands/sto_manager.js (hunk 1 above)
Comment (shuffledex) [reply CLI/commands/sto_manager.js]: ok

**Review by VictorVicente — APPROVED** [review]

**Pull request merged**